airflow/providers/google/cloud/sensors/dataflow.py

```diff
@@ -194,27 +246,72 @@ def poke(self, context: Context) -> bool:
             project_id=self.project_id,
             location=self.location,
         )
+        return result["metrics"] if self.callback is None else self.callback(result["metrics"])
+
+    def execute(self, context: Context) -> Any:
+        """Airflow runs this method on the worker and defers using the trigger."""
+        if not self.deferrable:
+            super().execute(context)
+        else:
```

Comment:
```suggestion
        elif not self.poke(context=context):
```

Reply:
Hi @e-galan , would it be possisble for us to change this way for this operator? Or is there anything I missed?